values/kubeapps/kubeapps.gotmpl

```diff
@@ -0,0 +1,8 @@
+{{- $v := .Environment.Values }}
+{{- $k := $v.charts | get "keycloak" dict }}
```

Comment:
If it cannot be integrated with keycloak, then how are we going to provide multi tenancy ?
I am missing some user story for this PR. Could provide it?

Reply:
It can be integrated with keycloak, but you have to finish jwt-to-rbac for that. This delivery is not dependent on that though.

Reply:
But what is the user story for Kubeapps. Who is using it, how it it accessed?